internal/ingress/controller/nginx.go

```diff
@@ -263,7 +263,7 @@ func (n *NGINXController) Start() {
 
 	// we need to use the defined ingress class to allow multiple leaders
 	// in order to update information about ingress status
-	// TODO: For now, as the the IngressClass logics has changed, is up to the
+	// TODO: For now, as the IngressClass logics has changed, is up to the
```

Comment:
`logics has` is not right. Is it possible to change that to `logic has`.

Reply:
```suggestion
	// TODO: For now, as the IngressClass logic has changed, is up to the
```